scio-core/src/main/scala/com/spotify/scio/util/Functions.scala

```diff
@@ -195,4 +195,14 @@ private[scio] object Functions {
     override def reduceOption(accumulator: Iterable[T]): Option[T] = _mon.sumOption(accumulator)
   }
 
+  /** Helper trait to decorate anonymous functions with meaningful toString. */
```

Comment:
@nevillelyh Small style question, would you prefer these stay here or move to another file `PrettyFn.scala`? I sort of prefer moving them so they're more visible now that they're being used across the package.